things/database.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, filepath=None):
     def get_tasks(
         self,
         uuid=None,
-        type="task",
+        type=None,
```

Comment:
while we're at it: the linters are not happy with a variable called `type`...

Reply:
In my opinion, that's an ok a usage. We are prioritizing convenience of API usage and consistency of naming with the Things app here. Yes, it's true, there is a built-in function called `type`, but in my opinion it's also fine to overwrite things in Python when the local usage is a more fitting one.

Examples from the standard library: [socket](https://github.com/python/cpython/blob/c4d45ee670c09d4f6da709df072ec80cb7dfad22/Lib/socket.py#L220), [asyncio](https://github.com/python/cpython/blob/e3ef4d7f653976ac0ccacc4e3fde06bf0e0f139b/Lib/asyncio/events.py#L295), [argparse](https://github.com/python/cpython/blob/e3ef4d7f653976ac0ccacc4e3fde06bf0e0f139b/Lib/argparse.py#L820), [tokenize](https://github.com/python/cpython/blob/e3ef4d7f653976ac0ccacc4e3fde06bf0e0f139b/Lib/tokenize.py#L45)